doc/md_src_files/040_stakeholders-scope.md

```diff
@@ -150,9 +150,9 @@ The [*Mission Impact*](#mission-impact) could be increased when a disaster recov
 A mitigation that successfully changes the value of a decision point may shift the priority of further action to a reduced state. If applying a mitigation reduces the priority to *defer*, the deployer may not need to apply a remediation, if later, it becomes available. Table 3 displays the action priorities for the deployer, which are similar to the supplier case.
 
 In a later section, the different types of impacts are defined and then implemented in the decision trees as examples of how the various impacts affect the priority.
-For now, assume the decision points are ordered as: [*Exploitation*](#exploitation); [*Exposure*](#exposure); [*Utility*](#utility); and Well-being and Mission Impact.
+For now, assume the decision points are ordered as: [*Exploitation*](#exploitation); [*Exposure*](#exposure); [*Utility*](#utility); and [*Well-being and Mission Impact*](#situated-safety---mission-impact).
```

Comment:
Should `Well-being and Mission Impact` also be changed to point to [*Human Impact*](#human-impact)?